app/components/govuk_component/back_link.rb

```diff
@@ -0,0 +1,13 @@
+class GovukComponent::BackLink < ViewComponent::Base
+  attr_accessor :text, :href, :options
+
+  def initialize(text:, href:, classes: nil, attributes: nil)
+    @text = text
+    @href = href
+    @classes = "govuk-back-link"
+    @classes << " #{classes}" if classes.present?
```

Comment:
Would it make sense to be able to pass in an array of classes?

Reply:
I think either approach can work fine, but possibly passing it as a string is less opinionated. I think most component libraries I've worked with expect strings, because that's what the core HTML elements themselves expect. Leave it to the caller to figure out how to make that string based on their requirements.

Reply:
What I like about either strings or array of strings is that they both should work with https://purgecss.com; as long as that is maintained, we should be good with either!

Reply:
Like @tvararu, I think most component libraries expect strings. I suppose if we really want to pass arrays we could check whether `classes` is an array or string and then build up the string internally. 

Either way I think we should be consistent with our approach because I think alot (if not all) components should support custom css classes like `govuk-frontend` does.